index.html

```diff
@@ -8,112 +8,138 @@
       <img class="soc-ico" src="./images/icon3.png" alt="twitter">
         </a>
       </li>
 
       <li>
-        <a class="social-icons" href="#">
+        <a class="social-icons sitop5" href="#">
       <img class="soc-ico" src="./images/icon4.png" alt="M">
          </a>
       </li>
     </ul>
+  </div>
+  </div>
+ 
     <!-- my recent works page -->
 
   <section class="second-page">
-    
+    <div class="recent-div">
+    <div class="second-page-div">
     <h2 class="second-heading">My Recent Works</h2>
-    <hr>
-
+    <hr class="page2-line">
+  </div>
     <!-- image  one for second page -->
 
     <a class="second-background">
-      <img src="./images/second-background.png" alt="background-image">
+      <img class="background2" src="./images/second-background.png" alt="background-image">
     </a>
 
     <!-- multi post stories section -->
+    <div class="page2-para">
+    <div class="recent-works-div">
 
     <h3 class="recent-work-heading">Multi-Post Stories</h3>
 
-    <p class="recent-work-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-
+    <p class="recent-work-para">A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+    
+    </div>
     <!-- languages cards  -->
 
-    <div>
       <ul class="languages">
         <li class="item1">css</li>
         <li class="item2">html</li>
         <li class="item3">bootsrap</li>
         <li class="item4">Ruby</li>
       </ul>
-    </div>
-
+    
     <!-- recent work submit button -->
 
     <button class="btn" type="button">
       <a class="button-text">See Project</a>      
     </button>
-  </section> 
+  </div>
+</div>
 
      <!-- professional art cards section -->
-
+     
      <section class="professional-art">
+      <div class="pro-div">
       <!-- First professional skill card -->
 
       <div id="pro-images-1" class="professional-art-text"> 
-        <h2 class="professional-text"> Profesional Art Printing Data </h2>
-        <p class="pro-para-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <h2 class="professional-text pro-text"> Profesional Art Printing Data </h2>
+        <p class="pro-para-text pro-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
         <!-- skill button in professional sectiion -->
       <ul>
-        <li class="card1">html</li>
-        <li class="card2">bootsrap</li>
-        <li class="card3">Ruby</li>
+        <li class="card1 car1">html</li>
+        <li class="card2 car1">bootsrap</li>
+        <li class="card3 car1">Ruby</li>
       </ul> 
```

Comment:
- [ ] Lastly for the project cards, when you see the figma design's project cards section you will notice the first card has the see project button but the rest don't that is because the first card is on `:hover` state (notice the different border color around it) :point_down: 
![image](https://user-images.githubusercontent.com/97521557/228860546-1478d492-403f-496b-bb58-af13fa33c0d3.png)

now, please notice that the other project cards do not have a see project button and so how would the user be able to these projects, like in the current state, so to fix this what you can do is implement a `:hover` state on your cards and on `:hover` you can make the opacity of description and title of the card 0 or low and make the see project button visible, or you can just make the see project button visible on `:hover` , however you want to implement but just fix this one issue and you are good to go :+1: